### PR TITLE
add blog post for the release of hpy-0.0.2

### PR DESCRIPTION
**Pull request by antocuni**
preview: https://deploy-preview-20--festive-wing-250462.netlify.app/blog/posts/2021/07/hpy-0.0.2-first-public-release/

### CODE REVIEW

File: posts/2021/07/hpy-0.0.2.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+.. date: 2021-07-19 10:00:00 UTC
+.. author: antocuni
+.. tags:
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+hpy 0.0.2 is out! This is the first version which is officially released and
```
Comment (hodgestar) [inline posts/2021/07/hpy-0.0.2.md:13]:
```suggestion
HPy 0.0.2 is out! This is the first version which is officially released and
```

File: posts/2021/07/hpy-0.0.2.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+which is used to autogenerate parts of the HPy code, is a reliable list of all
+the supported functions.
+
+!!! Warning
+
+    The HPy API is still considered in alpha status and it's subject to change
+    between versions. In fact, the current master is already incompatible with
+    hpy-0.0.2 because of
+    [PR #182](https://github.com/hpyproject/hpy/pull/182), which renamed all
+    occurences of `HPyContext` into `HPyContext*`.
```
Comment (hodgestar) [inline posts/2021/07/hpy-0.0.2.md:91]:
```suggestion
    occurences of `HPyContext` into `HPyContext *`.
```
Bit unsure whether with or without the space is more common / readable, so feel free to ignore the suggestion.

**Review by hodgestar — APPROVED** [review]
It needs the dates and versions filled in and I left a few tiny suggestions, but looks great to me. I especially like the examples section.

File: posts/2021/07/hpy-0.0.2.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+    Currently, we provide only the `sdist` (i.e., the `.tar.gz`, no binary
+    wheels). See also
+    [issue #223](https://github.com/hpyproject/hpy/issues/223), contributions
+    are welcome :).
+
+
+PyPy and GraalPython ships their own version of HPy, so no installation is
+necessary. HPy 0.0.2 will be included in the next release of both, i.e. PyPy
+x.x.x (expected on 2021-xx-xx) and GraalPython y.y.y (expected on
```
Comment (timfel) [inline posts/2021/07/hpy-0.0.2.md:49]: GraalPython 21.2.0 (expected on 2021-07-20

Comment (mattip) [reply posts/2021/07/hpy-0.0.2.md:49]: PyPy 7.3.6 expected in 10-2021

File: posts/2021/07/hpy-0.0.2.md (hunk 2 above)
Comment (antocuni) [reply posts/2021/07/hpy-0.0.2.md:91]: I tried with the space at first but I think it was too easy to overlook it when reading the rendered page

**Pull request merged**